shared/constants.ts

```diff
@@ -33,13 +33,6 @@ export const isSameVC = (vcKey: string, pinnedVcKey: string) => {
   return requestId === pinnedRequestId;
 };
 
-//To compare the vckey with requestId, when the vc is pinned
-export const isSameVC = (vcKey: string, pinnedVcKey: string) => {
```

Comment:
why removing this method?

Reply:
got it. added twice

Reply:
This method is declared twice with same content. Thus making build fail
Removed this to avoid build failure @swatigoel